.github/workflows/testPublish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x]
+        node-version: [18.x, 20.x]
```

Comment:
Why are we doing 18 and 20 here but changed to only 20 on line 40?

Reply:
@blackfalcon , could you please explain as this code was changed by you?

Reply:
This is all explained here https://auro.alaskaair.com/node-support

We are testing. for 18 and 20, but there is no reason to retest for 18 again with the release. The release is focused on the current LTS.